src/gei/Commands/MigrateRepo/MigrateRepoCommandArgs.cs

```diff
@@ -61,6 +62,15 @@ public override void Validate(OctoLogger log)
                 {
                     throw new OctoshiftCliException("--ghes-api-url must be specified when --keep-archive is specified.");
                 }
+                if (UseGithubStorage)
+                {
+                    throw new OctoshiftCliException("--ghes-api-url must be specified when --use-github-storage is specified.");
+                }
+            }
+
+            if (AwsBucketName.HasValue() && UseGithubStorage)
+            {
+                throw new OctoshiftCliException("The --use-github-storage flag was provided with an AWS S3 Bucket name. Archive cannot be uploaded to both locations.");
```

Comment:
I guess we also need a check for when `--use-github-storage` is provided with `AzureStorageConnectionString`